src/app/tabs/run-tab.js

```diff
@@ -264,12 +274,12 @@ function contractDropdown (appAPI, appEvents, instanceContainer) {
       </div>
       <div class="${css.buttons}">
         <div class="${css.button}">
-          <div class="${css.atAddress}" onclick=${function () { loadFromAddress(appAPI) }}>At Address</div>
-          ${atAddressButtonInput}
+        ${createButtonInput}
+        <div class="${css.create}" onclick=${function () { createInstance() }} >Create</div>
         </div>
         <div class="${css.button}">
-          <div class="${css.create}" onclick=${function () { createInstance() }} >Create</div>
-          ${createButtonInput}
+          ${atAddressButtonInput}
+          <div class="${css.atAddress}" onclick=${function () { loadFromAddress(appAPI) }}>Load</div>
```

Comment:
I m not sure `Load` is the right word for that... that can be confusing ... perhaps `Watch` would be better

Reply:
I will switch it back to atAddress for now and since there's a plan to create this dropdown where user can choose whether they want to create or 'load/watch/see existing...' we can then rethink this again.